include/pstore/exchange/export_paths.hpp

```diff
@@ -0,0 +1,36 @@
+//===- include/pstore/exchange/export_paths.hpp -----------*- mode: C++ -*-===//
+//*                             _                 _   _          *
+//*   _____  ___ __   ___  _ __| |_   _ __   __ _| |_| |__  ___  *
+//*  / _ \ \/ / '_ \ / _ \| '__| __| | '_ \ / _` | __| '_ \/ __| *
+//* |  __/>  <| |_) | (_) | |  | |_  | |_) | (_| | |_| | | \__ \ *
+//*  \___/_/\_\ .__/ \___/|_|   \__| | .__/ \__,_|\__|_| |_|___/ *
+//*           |_|                    |_|                         *
+//===----------------------------------------------------------------------===//
+//
+// Part of the pstore project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://github.com/SNSystems/pstore/blob/master/LICENSE.txt for license
+// information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+/// \file export_paths.hpp
+/// \brief  Exporting the members of the paths index.
+
+#ifndef PSTORE_EXCHANGE_EXPORT_PATHS_HPP
+#define PSTORE_EXCHANGE_EXPORT_PATHS_HPP
+
+#include "pstore/exchange/export_names.hpp"
```

Comment:
Yes, I considered doing this. I decided against for the moment because it will also induce a bunch of /name/string/ renaming. It would also make sense to change import_names.* at the same time. That would be an awful lot of noise to add to this change.